### PR TITLE
- Fix geonode project dependencies and fixtures [rev]

### PR DESCRIPTION
**Pull request by afabiani**

### CODE REVIEW

File: project_name/settings.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 #########################################################################
 #
-# Copyright (C) 2016 OSGeo
+# Copyright (C) 2017 OpenPlans
```
Comment (kalxas) [inline project_name/settings.py]: The copyright has moved to OSGeo...

File: project_name/settings.py [hunk 2]
```diff
@@ -98,1113 +56,26 @@
...
-    ABSOLUTE_URL_OVERRIDES = os.getenv('ABSOLUTE_URL_OVERRIDES',_DEFAULT_ABSOLUTE_URL_OVERRIDES)
-    AUTH_PROFILE_MODULE = os.getenv('AUTH_PROFILE_MODULE','maps.Contact')
-    REGISTRATION_OPEN =  strtobool(os.getenv('REGISTRATION_OPEN', 'True'))
-
-    ACCOUNT_ACTIVATION_DAYS = int(os.getenv('ACCOUNT_ACTIVATION_DAYS','7'))
-
-    # TODO: Allow overriding with an env var
-    DB_DATASTORE = strtobool(os.getenv('DB_DATASTORE', 'True'))
-
-    ALLOWED_HOSTS = os.getenv('ALLOWED_HOSTS', ['localhost', ])
```
Comment (kalxas) [inline project_name/settings.py:1194]: We might need to keep ALLOWED_HOSTS in here so that users avoid changing it in the geonode.settings

Comment (afabiani) [reply project_name/settings.py]: Fixed

File: project_name/settings.py (hunk 1 above)
Comment (afabiani) [reply project_name/settings.py]: Fixed

File: project_name/settings.py (hunk 2 above)
Comment (afabiani) [reply project_name/settings.py:1194]: Fixed

**Pull request merged**